### PR TITLE
Use local registry for local bundles in catalog build

### PR DESCRIPTION
**Pull request by rabi**
This would help bundles built with PR from local registry.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

### CODE REVIEW

**Comment by rabi** [comment]
recheck

**Comment by rabi** [comment]
recheck

**Review by raukadah — APPROVED** [review]
/approve
/lgtm

File: ci_framework/roles/operator_build/tasks/build.yml [hunk 1]
```diff
@@ -157,10 +157,6 @@
     params:
       IMG: "{{ operator_img }}"
       BUNDLE_IMG: "{{ operator_img_bundle }}"
-      IMAGENAMESPACE: "{{ cifmw_operator_build_push_org }}"
```
Comment (Sandeepyadav93) [inline ci_framework/roles/operator_build/tasks/build.yml:160]: we need those image var for bundle and catalog_build both targets only and not bundle_build as openstack-operator bundle does not include other csvs now

File: ci_framework/roles/operator_build/tasks/build.yml [hunk 2]
```diff
@@ -146,7 +146,7 @@
       IMAGENAMESPACE: "{{ cifmw_operator_build_push_org }}"
       IMAGEREGISTRY: "{{ cifmw_operator_build_push_registry }}"
       IMAGEBASE: "{{ operator.image_base | default('') }}"
-      LOCAL_REGISTRY: "{{ cifmw_operator_build_local_registry | default ('') }}"
```
Comment (Sandeepyadav93) [inline ci_framework/roles/operator_build/tasks/build.yml:149]: looks good

script expect an integer
https://github.com/openstack-k8s-operators/openstack-operator/blob/main/hack/pin-bundle-images.sh#L47

**Pull request merged**